microsoft-edge/webview2/concepts/distribution.md

```diff
@@ -1,150 +1,280 @@
+    *   In the setup part of your WebView2 app, link to the bootstrapper.  Use a link to programmatically download the bootstrapper; select the **Get the Link** button at the above download page.
+    *   Or, download the bootstrapper and package it with your WebView2 app.
 
-*   WebView2 Runtime Bootstrapper is a tiny \(approximately 2 MB\) installer.  WebView2 Runtime Bootstrapper downloads and installs the Evergreen Runtime from Microsoft servers that matches the user's device architecture.  
-*   Use a link to programmatically download the bootstrapper.  
-*   WebView2 Runtime Standalone Installer is a full installer that installs the Evergreen WebView2 Runtime in offline environments.  
+*   For offline clients: _WebView2 Runtime Standalone Installer_ is a full installer that installs the Evergreen WebView2 Runtime in offline environments.
     
-Currently, both the bootstrapper and standalone installer only support per-machine installs, which requires elevation.  If an installer is run without elevation, the user is prompted to elevate permissions.  
+Currently, both the bootstrapper and standalone installer only support per-machine installs, which requires elevation of permissions.  If an installer is run without elevation, the user is prompted to elevate permissions.  
 
-Use following workflows to ensure the Runtime is already installed before your app launches.  You may adjust your workflow depending on your scenario.  Sample code is available in the [Samples repo][GitHubMicrosoftedgeWebView2samplesWebview2Deployment].  
+Use the following online deployment workflow or offline deployment workflow to ensure that the Runtime is already installed before your app launches.  You can adjust your workflow depending on your scenario.  Sample code is available in the [Samples repo][GitHubMicrosoftedgeWebView2samplesWebview2Deployment].  
 
 #### Online-only deployment  
 
 If you have an online-only deployment scenario where users are assumed to have internet access, complete the following steps.  
 
-1.  During your app setup, ensure the Runtime is already installed.  To  verify, complete one of the following actions.  
-    *   Inspect if the `pv (REG_SZ)` regkey exists and is not `null` or empty.  Find  `pv (REG_SZ)` at the following location.  
+1.  During your app setup, run a test to make sure that the WebView2 Runtime is already installed.  To verify that the Runtime is installed, use either of the following approaches:
+
+    *   Inspect the `pv (REG_SZ)` registry key for the WebView2 runtime at the following location.  If this regkey doesn't exist, or if exists and is `null` or an empty string, this means that the WebView2 Runtime is not installed on the client.  Find `pv (REG_SZ)` at the following location.  
```

Comment:
@liminzhu - Pls check this logic of how to check regkey, and what to do with `GetAvailableCoreWebView2BrowserVersionString` result, in this section and "offline" section below.